cli/tests/unit/metrics_test.ts

```diff
@@ -76,3 +76,14 @@ Deno.test(function metricsForOpCrates() {
   assert(m1.opsDispatched > 0);
   assert(m1.opsCompleted > 0);
 });
+
+// Test that op_names == Objects.keys(Deno.core.ops)
+// since building the per-op metrics depends on op_names being complete
+Deno.test(function opNamesMatch() {
+  assertEquals(
+    // @ts-ignore: Deno.core allowed
+    Deno.core.opNames().sort(),
+    // @ts-ignore: Deno.core allowed
+    Object.keys(Deno.core.ops).sort(),
```

Comment:
Why do we need `Deno.core.opNames()` if you can get names of ops via `Deno.core.ops`?

Reply:
The order of op names from `opNames()` aligns with the order of metrics (essentially idx == op_id), the `Object.keys(Deno.core.ops)` doesn't preserve order so we can't form the correct pairs